bin/deploy-vhost

```diff
@@ -187,6 +187,9 @@ if ($conf->{redirects_only}) {
 
 # Make database config settings
 if (defined $conf->{'databases'}) {
+    # Empty arrays for the structured configurations
+    $conf->{struct_database_configs}{internal} = ();
```

Comment:
Note, though, that perl will auto create the array refs when you push below, so there's not actually a need to create an empty one here if you don't want. The other ones e.g. `default` don't create themselves beforehand.